qlib/portfolio/optimizer/optimizer.py

```diff
@@ -1,15 +1,17 @@
 # Copyright (c) Microsoft Corporation.
 # Licensed under the MIT License.
 
+
 import warnings
 import numpy as np
 import pandas as pd
 import scipy.optimize as so
-
 from typing import Optional, Union, Callable, List
 
+from qlib.portfolio.optimizer import BaseOptimizer
+
 
-class PortfolioOptimizer:
+class PortfolioOptimizer(BaseOptimizer):
```

Comment:
We may need to split this optimizer into `mean_variance.py`, `risk_parity.py`, etc and at least use `cvxpy` as the solver for `mean_variance`.

Reply:
After discussion, we decided to save those changes to the next pull request.